page-components/form-fields.md

```diff
@@ -1106,7 +1108,7 @@ Target area
 
 Radio button (or checkbox)
 
-- Border: 1 px, Pacific (#0072ce)
+- Border: 1 px, Gray 60 (#919395)
```

Comment:
This change is incorrect, I believe. The [live example in this location](https://cfpb.github.io/design-manual/page-components/form-fields.html#selected-2) shows a border that is definitely Pacific.

Reply:
This refers to the circle around the radio button, not the rectangle around the whole thing.

<img width="34" alt="Screen Shot 2019-07-01 at 5 31 50 PM" src="https://user-images.githubusercontent.com/704760/60467975-20883b00-9c26-11e9-819a-851f1ae0f48e.png">

Reply:
Oh, that seems so obvious now 🤦‍♂